server/src/test/java/io/crate/integrationtests/CopyIntegrationTest.java

```diff
@@ -861,8 +862,8 @@ public void test_copy_from_with_fail_fast_property_with_one_node_configuration()
                          "data1.json",
                          target);
 
-
         execute("CREATE TABLE t (a int)");
+        ensureGreen();
```

Comment:
Why did the `ensureGreen` calls become necessary? They shouldn't be required

Reply:
I had a weird behaviour that resolved by having ensureGreen(), but seems gone. Taking all out and merging, Thank you.